README.rst

```diff
@@ -63,3 +63,11 @@ to Sentry:
 This is helpful if you want to report errors to users but not to Thoth
 application itself.
 
+
+If you want some certain classes of error to be not reported to Sentry. You can
+can provide ``THOTH_SENTRY_IGNORE_EXCEPTION`` environment variable which holds
+the comma separated list of exception classes to be ignored when reporting to Sentry:
+
+.. code-block:: console
+
+  THOTH_SENTRY_IGNORE_EXCEPTION="connexion.decorators.validation,ValueError"
```

Comment:
Is the comma here intentional? Is it delimiting module and exception class or is it just a typo?

Reply:
It's there to facilitate a split between different exception classes.

Reply:
So one exception is `connexion.decorators.validation` and another one is `ValueError`?

Reply:
Yes

Reply:
please adjust `ValueError` to `builtins.ValueError` in that case.